lib/core/errors.js

```diff
@@ -124,23 +138,27 @@ class AbortError extends UndiciError {
     return instance && instance[kAbortError] === true
   }
 
-  [kAbortError] = true
+  get [kAbortError] () {
+    return true
+  }
 }
 
-const kRequestAbortedError = Symbol.for('undici.error.UND_ERR_ABORTED')
+const kRequestAbortedError = Symbol.for('undici.error.UND_ERR_REQUEST_ABORTED')
 class RequestAbortedError extends AbortError {
   constructor (message) {
     super(message)
-    this.name = 'AbortError'
+    this.name = 'RequestAbortedError'
```

Comment:
Just as a warning, changing them imposes a breaking change; so we might need to split the changes so we can land this on minor and upgrade later for major